_posts/2017-11-03-adapter-model.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Mixer Adapter Model"
+overview: Provides an overview of Istio 0.2's updated Mixer adapter model
+published: true
+permalink: blog/mixer-adapter-model.html
+attribution: The Istio Team
+layout: post
+type: markdown
+---
+{% include home.html %}
+
+Istio 0.2 introduced a new Mixer adapter model which is intended to increase Mixer’s flexibility to address a varied set of infrastructure backends. This post intends to put the adapter model in context and explain how it works.
+
+<!--end_excerpt-->
+
+## Why Adapters?
+
+Infrastructure backends provide support functionality used to build services. They include such things as access control systems, telemetry capturing systems, quota enforcement systems, billing systems, and so forth. Services traditionally directly integrate with these backend systems, creating a hard coupling and baking-in specific semantics and usage options.
+
+Mixer serves as an abstraction layer between Istio and an open-ended set of infrastructure backends. The Istio components and services that run within the mesh can interact with these backends, while not being coupled to the backends’ specific interfaces.
+
+In addition to insulating application-level code from the details of infrastructure backends, Mixer provides an intermediation model that allows operators to inject and control policies between application code and backends. Operators can control which data is reported to which backend, which backend to consult for authorization, and much more.
+
+Given that individual infrastructure backends each have different interfaces and operational models, Mixer needs custom code to deal with each and we call these custom bundles of code *[adapter*s](https://github.com/istio/istio/blob/master/mixer/doc/dev/adapters.md).
```

Comment:
I feel like the '*'s surrounding adapter aren't doing what is intended.  Here and elsewhere (templates, instances, rules), should this be: `[*adapter*s]` (instead of `*[adapter*s]`) ?